src/amp-story-player/amp-story-player-impl.js

```diff
@@ -211,6 +216,26 @@ export class AmpStoryPlayer {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /**
+   * Helper to create a button.
+   * @private
+   */
+  initializeButton_() {
+    const option = this.element_.getAttribute('button');
+    if (!Object.values(BUTTON_TYPES).includes(option)) {
+      return;
+    }
+
+    const button = this.doc_.createElement('a');
+    button.classList.add('amp-story-player-' + option + '-button');
+
+    button.addEventListener('click', () => {
+      this.element_.dispatchEvent(createCustomEvent(this.win_, option, {}));
```

Comment:
This will dispatch events with the name `back` or `close`, but I think we might want them to be more specific (e.g. `amp-story-player-back`), so that our events don't collide with other events that could be triggered on the page.

WDYT @connielei @Enriqe?

Reply:
He mentioned that above. That makes sense!